woodwork/table_accessor.py

```diff
@@ -84,70 +113,81 @@ def init(self,
-            schema (Woodwork.TableSchema, optional): Typing information to use for the DataFrame instead of performing inference.
-                Any other arguments provided will be ignored. Note that any changes made to the schema object after
-                initialization will propagate to the DataFrame. Similarly, to avoid unintended typing information changes,
-                the same schema object should not be shared between DataFrames.
             validate (bool, optional): Whether parameter and data validation should occur. Defaults to True. Warning:
                 Should be set to False only when parameters and data are known to be valid.
                 Any errors resulting from skipping validation with invalid inputs may not be easily understood.
         """
-        if validate:
-            _validate_accessor_params(self._dataframe, index, time_index, logical_types, schema, use_standard_tags)
-        if schema is not None:
-            self._schema = schema
-            extra_params = []
-            if index is not None:
-                extra_params.append('index')
-            if time_index is not None:
-                extra_params.append('time_index')
-            if logical_types is not None:
-                extra_params.append('logical_types')
-            if already_sorted:
-                extra_params.append('already_sorted')
-            if not use_standard_tags or isinstance(use_standard_tags, dict):
-                extra_params.append('use_standard_tags')
-            for key in kwargs:
-                extra_params.append(key)
-            if extra_params:
-                warnings.warn("A schema was provided and the following parameters were ignored: " + ", ".join(extra_params), ParametersIgnoredWarning)
-
-        else:
-            # Perform type inference and update underlying data
-            parsed_logical_types = {}
-            for name in self._dataframe.columns:
-                series = self._dataframe[name]
-
-                logical_type = None
-                if logical_types:
-                    logical_type = logical_types.get(name)
-
-                logical_type = _get_column_logical_type(series, logical_type, name)
-                parsed_logical_types[name] = logical_type
-
-                updated_series = logical_type.transform(series)
-                if updated_series is not series:
-                    self._dataframe[name] = updated_series
-
-            column_names = list(self._dataframe.columns)
-
-            # TableSchema uses a different default for use_standard_tags so we need to define it here
+        def left_join(left: dict, right: dict):
+            left = left or {}
+            right = right or {}
+            combined_dict = {}
+            for key in right:
+                combined_dict[key] = right[key]
+            for key in left:
+                combined_dict[key] = left[key]
+            return combined_dict
```

Comment:
if I'm understanding the goal for this function correctly, you should be able to do `{**right, **left}`, and that will create a new dictionary object that will contain the `right` and `left` items, with any duplicate keys giving preference to `left`